sdk-react/src/useAnimaCodegen.ts

```diff
@@ -289,6 +289,8 @@ export const useAnimaCodegen = ({
           if (
             errorPayload?.payload?.name === "Task Crashed" ||
             errorPayload?.payload?.name === "TimeoutError" ||
+            errorPayload?.payload?.name === "Error" ||
+            errorPayload?.payload?.name === "Unknown error" ||
             errorStatusCode === 429
```

Comment:
In this case, I think it would be better to have a whitelist for which errors we can retry.
But I'm okay with this approach.

Reply:
You're right, I didn't do it already as a whitelist as I first need to map out all the network error cases :) I'll try to do it in a follow up PR, this is mostly a quick win